packages/front-end/components/Features/EditFeatureInfoModal.tsx

```diff
@@ -83,7 +86,7 @@ const EditFeatureInfoModal: FC<{
             value={form.watch("project")}
             onChange={(v) => {
               form.setValue("project", v);
-              setShowProjectWarningMsg(true);
+              setShowProjectWarningMsg(v !== feature.project);
```

Comment:
To fix a small UI bug where if I selected the same project the warning would show when in reality nothing was changing.